_docs/_latest/setup/manual-gcp-deployment.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Manual Deployment
+order: 004
+---
+
+# {{ page.title }}
+
+This page describes the steps to install and deploy Forseti manually on 
+Google Cloud Platform (GCP). It's best to use the
+[automated installer]({% link _docs/latest/setup/install.md %})
+for an easy and error-free deployment. Use this guide only if you strongly
+prefer to install Forseti yourself, or if you want to learn about the
+installation process.
+
+To complete this guide, you will need in-depth knowledge of GCP. For example,
+you will need to know how to use the gcloud command-line tool and where to bind
+a service account to a VM in the GCP Console. This guide refers to the installer
+and deployment templates for specific details of the commands to use.
+
+## Create a project
+
+Create a new project to host Forseti.  Forseti is intended to run in its own
+dedicated project so that access to its highly privileged permissions can be
+controlled.  Assign a billing account to the project.
+
+### Enable APIs
+
+For information about the APIs you need to enable in the project that hosts
+Forseti, see [required APIs]({% link _docs/latest/required_apis.md %}).
+
+## Deploy a server VM
+
+### Create a Forseti server service sccount
+
+Create a server service account by running the following command:
```

Comment:
Forgo use of the ":" in these types of statements.

Reply:
I disagree with that, we really should have appropriate punctuation. What's the impetus for removing it?

Reply:
We weren't always using it correctly or consistently. We can leave it, but we'll just want need to revisit the other docs.

Reply:
Gotcha, I'm happy to help with a scrub at some point.